test/tools/TestMetadata.json

```diff
@@ -1,6 +1,7 @@
 {
   "ExperimentalFeatures": {
     "ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1" ],
-    "PSForEachObjectParallel": [ "test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1", "test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1" ]
+    "PSForEachObjectParallel": [ "test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1", "test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1" ],
+    "Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints": ["test/powershell/Modules/Microsoft.PowerShell.Utility/New-PSBreakpoint.Tests.ps1"]
```

Comment:
Merge in the latest changes from master. That experimental feature was only ever partially implemented, and has been replaced with something else.

Reply:
Yes, see `Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace` instead. Experimental features are turned on by default for preview/private builds, so no need to add to this file.
Changes to this file should be reverted.

Reply:
Looks like my 2nd rebase didn't get the change in https://github.com/PowerShell/PowerShell/commit/cc0fed479a3e455b746a3d12597f078462f2d644#diff-486e12b35a01f2cd00589c5293d83f1c

Reply:
Do we ever need to run tests when this isn't the case, like for stable builds?

Reply:
Good point. For experimental features that are not made official at GA time, their tests should be declared here so that the tests get exercised when running tests on the stable builds.
For that reason, it makes sense to keep the experimental tests here.